models/pair.go

```diff
@@ -45,7 +45,11 @@ func NewPairMngr() *PairMngr {
 	m := PairMngr{
 		aToken2PairArr: make(map[string][]*Pair),
 	}
-	m.aToken2PairArr["ETH"] = []*Pair{NewPair("ETH", "USD")}
+	symbolPairs := GetAllSymbols()
+	for _, sp := range symbolPairs {
```

Comment:
This needs refactoring. I think you already are aware. Waste of compute doing this loop everytime

Reply:
yes, when we will refactor symbols/pairs/tokens